tests/webhook_test.go

```diff
@@ -159,15 +158,19 @@ var _ = Describe("Validation webhook", func() {
 
 		Context("Placement API validation", func() {
 			It("[test_id:5990]should succeed with valid template-validator placement fields", func() {
-				foundSsp.Spec.TemplateValidator.Placement = &placementAPIValidationValidPlacement
-				Expect(apiClient.Update(ctx, foundSsp, client.DryRunAll)).ToNot(HaveOccurred(),
-					"failed to update SSP CR with valid template-validator placement fields")
+				Eventually(func() error {
+					foundSsp = getSsp()
+					foundSsp.Spec.TemplateValidator.Placement = &placementAPIValidationValidPlacement
+					return apiClient.Update(ctx, foundSsp, client.DryRunAll)
+				}, time.Second, tenSecondTimeout).ShouldNot(HaveOccurred(), "failed to update SSP CR with valid template-validator placement fields")
 			})
 
 			It("[test_id:5989]should fail with invalid template-validator placement fields", func() {
-				foundSsp.Spec.TemplateValidator.Placement = &placementAPIValidationInvalidPlacement
-				Expect(apiClient.Update(ctx, foundSsp, client.DryRunAll)).To(HaveOccurred(),
-					"SSP CR updated with invalid template-validator placement fields")
+				Eventually(func() error {
+					foundSsp = getSsp()
+					foundSsp.Spec.TemplateValidator.Placement = &placementAPIValidationInvalidPlacement
+					return apiClient.Update(ctx, foundSsp, client.DryRunAll)
+				}, time.Second, tenSecondTimeout).Should(HaveOccurred(), "SSP CR updated with invalid template-validator placement fields")
```

Comment:
Can you check for specific error here? Like this:
https://github.com/kubevirt/ssp-operator/blob/master/tests/validator_test.go#L797-L798